### PR TITLE
feat: meeting started webhook

### PR DESCRIPTION
**Pull request by ibex088**
/claim #12728

right now the output format of the webhook is similar to meeting-ended, but different from all other webhook.
 Additionally, it doesn't currently sign the header with the webhook secret.

### CODE REVIEW

File: packages/features/webhooks/lib/scheduleTrigger.ts [hunk 1]
```diff
@@ -45,7 +45,10 @@ export async function addSubscription({
       },
     });
 
-    if (triggerEvent === WebhookTriggerEvents.MEETING_ENDED) {
+    if (
+      triggerEvent === WebhookTriggerEvents.MEETING_ENDED
+      // || triggerEvent === WebhookTriggerEvents.MEETING_STARTED
```
Comment (SomayChauhan) [inline packages/features/webhooks/lib/scheduleTrigger.ts:50]: not sure about this

Comment (CarinaWolli) [reply packages/features/webhooks/lib/scheduleTrigger.ts:50]: I think this is needed for zapier and make.com. As we mostly likely add that trigger there too one day, we can already add this here as well 👍

**Review by CarinaWolli — COMMENTED** [review]
Nice work 👏🏻 Code looks good, can you uncomment this line of code: https://github.com/calcom/cal.com/pull/12764/files#diff-9ca4b590fb0eb3a0c0be9708bfa9afea092aec81cacac8187fc63864d426fdf9R50

Then it's good to go :)

File: apps/web/public/static/locales/en/common.json [hunk 2]
```diff
@@ -2160,5 +2160,6 @@
   "need_help": "Need help?",
   "troubleshooter": "Troubleshooter",
   "please_install_a_calendar": "Please install a calendar",
+  "meeting_started": "Meeting Started",
```
Comment (Aravindh09) [inline apps/web/public/static/locales/en/common.json:2163]: Suggestion : "meeting_started": "Meeting started", - "started" to start with lower case "s"

Comment (CarinaWolli) [reply apps/web/public/static/locales/en/common.json:2163]: It should be uppercase like all the other webhooks as well: 
Screenshot 2023-12-19 at 8 47 24 AM

**Review by CarinaWolli — APPROVED** [review]
Great work @SomayChauhan 👏🏻 Thank you 🙏

**Pull request merged**